### ISSUE TITLE
PDF build says failed when it has actually succeeded

### ISSUE THREAD

**Issue by astrofrog** [rtfd/readthedocs.org#1204]
For example, for the Astropy PDF build:

https://readthedocs.org/builds/astropy/2483122/

the final lines of output are:

```
Output written on Astropy.pdf (1407 pages, 7211162 bytes).
Transcript written on Astropy.log.
```

and the PDF file works, so not sure why it is being marked as failed.

**Comment by astrofrog** [comment]
Ah but somehow the PDF for that version doesn't exist:

https://media.readthedocs.org/pdf/astropy/v1.0.1/astropy.pdf

so there must be some error I am missing. @ericholscher - any ideas?

**Comment by gregmuellegger** [comment]
It seems that PDF build for your latest version (1.0.3) was reported as success and the PDF is available. @astrofrog is this still an issue for you?

**Comment by ericholscher** [comment]
This is fixed in #1524
